### ISSUE TITLE
Allow importcontent to change download timeout

### ISSUE THREAD

**Issue by dbnicholson** [learningequality/kolibri#9269]
## Observed behavior

When running the `importcontent` management command, we often get read timeouts downloading larger files.

## Errors and logs

```
ERROR    Error reading download stream: HTTPSConnectionPool(host='*****', port=443): Read timed out.
```

## Expected behavior

It would be nice to tell `importcontent` to wait longer than (AFAICT) the default 20 seconds in the `Transfer` class.

## User-facing consequences

If you're on a slow network, then you might get timeouts trying to import channel contents. Allowing a longer timeout might let those users import channel files more reliably.

## Steps to reproduce

It's hard to reproduce without being on a slow network or being able to inject a delay into the transfer. Trying to import a channel with large videos like Khan Academy is more likely to hit the timeouts, though.

```
python -m kolibri manage importchannel network c9d7f950ab6b5a1199e3d6c10d7f0103
python -m kolibri manage importcontent network c9d7f950ab6b5a1199e3d6c10d7f0103
```

## Context

We're using Kolibri via the `org.learningequality.Kolibri` flatpak app in flathub. Currently this uses kolibri-installer-gnome 2.0, which in turn includes Kolibri 0.14.7 (AFAICT). I believe soon that will be using 0.15.0. We have several products where we want to preload OS images with Kolibri content.

What we're trying to do is offload Learning Equality's CDN since we fetch a lot of content frequently. I've put another CDN in front of Learning Equality's CDN. Once our CDN has the file cached, there would be no connection back to LE's CDN. However, when there's a cache miss on our end, our CDN has to fetch from LE's CDN. In that case, Kolibri has to wait until the object has been fetched into our CDN's cache before it will begin streaming the output to Kolibri.

This can be slow for larger files. What I'd like to do is to turn the read timeout much higher like 5 minutes or something. This is an automated process, so we'd rather have it wait a while than fail to download the file. I would even venture to say that the default 20 seconds is too low. I'd expect something more like 1 minute to accommodate slower networks. But I'm fine leaving the default alone so long as I can set it higher as needed.

**Comment by rtibbles** [comment]
I think given that many of our end users might be running on slower or at least higher latency connections, there is probably some merit to increasing the default timeout to say, 60 seconds, and then also making it's configurable through the management command.

As this is already configurable on the transfer class https://github.com/learningequality/kolibri/blob/develop/kolibri/core/content/utils/transfer.py#L36 this should be relatively straightforward.

**Comment by dbnicholson** [comment]
Ok, I'll look at the default as well as the override.

For the override the only thing that I'm unsure about is the semantics. The simple way would be to duplicate the `Transfer` default as the option default. But I think more proper would be for the CLI not to pass the argument unless the option was specified. That would be a little uglier to code but not too bad.

**Comment by dbnicholson** [comment]
> I think given that many of our end users might be running on slower or at least higher latency connections, there is probably some merit to increasing the default timeout to say, 60 seconds, and then also making it's configurable through the management command.

#9281 increases the default to 60 seconds.

**Comment by rtibbles** [comment]
Fixed in #9289
